client/my-sites/plans-features-main/index.jsx

```diff
@@ -72,6 +77,116 @@ import {
 
 import './style.scss';
 
+const OnboardingPricingGrid2023 = ( { plans, visiblePlans, ...props } ) => {
```

Comment:
This is fine to live here for now AFAIC. We'll extract and type properly at a later point.